src/app/private/scripts/controllers/dashboard/orgDashboardController.js

```diff
@@ -224,5 +224,39 @@ angular.module('playerApp')
         dashboardData.orgId = orgId;
         dashboardData.getAdminDashboardData();
       };
+
+      /**
+       * @Function downloadReports
+       * @Description - make dowload csv api call
+       * @Return  void
+       */
+      dashboardData.downloadReport = function() {
+        dashboardData.showDownloadLoader = 'active';
+        var req = {
+          org_id: dashboardData.orgId,
+          period: dashboardData.timePeriod
+        };
+
+        // Call service
+        dashboardService.downloadReport(req, dashboardData.datasetPreviousValue).then(function(apiResponse) {
```

Comment:
this code block seems to be a duplicate of the code in course dashboard controller. please remove the copied code and use a shared implementation.